File: gnn_model/data/account.py
class Account:
    def __init__(self, account_id, bank, currency) -> None:
        self.account_id = account_id
        self.bank=bank
        self.currency=currency

    @classmethod
    def from_json(cls, json_data):
        return cls(
            account_id=json_data['accountId'],
            bank=json_data['bank'],
            currency=json_data['currency'])

File: graphaml/src/main/java/ru/uniserg/graphaml/data/Account.java
package ru.uniserg.graphaml.data;

public class Account {
    private Long accountId;
    private String currency;
    private String bank;

    public Account(Long accountId, String currency, String bank) {
        this.accountId = accountId;
        this.currency = currency;
        this.bank = bank;
    }

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }

    public String getBank() {
        return bank;
    }

    public void setBank(String bank) {
        this.bank = bank;
    }

    public Long getAccountId() {
        return accountId;
    }

    public void setAccountId(Long accountId) {
        this.accountId = accountId;
    }

    @Override
    public String toString() {
        return "Account{" +
                "accountId=" + accountId +
                ", currency='" + currency + '\'' +
                ", bank='" + bank + '\'' +
                '}';
    }
}
